lessons/computation-transformations/langs/en-us/index.adoc

```diff
@@ -0,0 +1,151 @@
+= Computation as a Circle Transformation
```

Comment:
Not sure what a Circle Transformation is or what this title is supposed to mean?